x-pack/plugins/uptime/public/components/overview/alerts/monitor_expressions/filters_expression_select.tsx

```diff
@@ -24,38 +23,35 @@ export const FiltersExpressionsSelect: React.FC<Props> = ({
   setAlertParams,
   newFilters,
   onRemoveFilter,
+  alertParams,
 }) => {
   const {
     filters: { tags, ports, schemes, locations },
   } = useSelector(overviewFiltersSelector);
 
-  const [updatedFieldValues, setUpdatedFieldValues] = useState<{
-    fieldName: string;
-    values: string[];
-  }>({ fieldName: '', values: [] });
-
-  const { selectedLocations, selectedPorts, selectedSchemes, selectedTags } = useFilterUpdate(
-    updatedFieldValues.fieldName,
-    updatedFieldValues.values
-  );
-
-  const [filters, setFilters] = useState<StatusCheckFilters>({
-    'observer.geo.name': selectedLocations,
-    'url.port': selectedPorts,
-    tags: selectedTags,
-    'monitor.type': selectedSchemes,
-  });
-
-  useEffect(() => {
-    setAlertParams('filters', filters);
-  }, [filters, setAlertParams]);
+  const selectedPorts = alertParams?.filters?.['url.port'] ?? [];
```

Comment:
I think we should store these as constants, not a priority for this PR, but something like this will be nice, APM is doing it like this https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

Reply:
Elastic search field names file.

Reply:
Created issue here: https://github.com/elastic/uptime/issues/219.